Mage.Sets/src/mage/cards/m/MonstrousStep.java

```diff
@@ -0,0 +1,114 @@
+package mage.cards.m;
+
+import java.util.UUID;
+import mage.abilities.Ability;
+import mage.abilities.effects.RequirementEffect;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.abilities.keyword.CyclingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetCreaturePermanent;
+import mage.watchers.common.BlockedAttackerWatcher;
+
+/**
+ *
+ * @author drowinternet
+ */
+public final class MonstrousStep extends CardImpl {
+
+    private static final FilterCreaturePermanent filterMustBlock = new FilterCreaturePermanent("Creature that must block");
+    private static final FilterCreaturePermanent filterToBeBlocked = new FilterCreaturePermanent("Creature that is to be blocked");
+
+    public MonstrousStep(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{4}{G}");
+
+
+        // Target creature gets +7/+7 until end of turn. Up to one target creature blocks it this turn if able.
+        this.getSpellAbility().addEffect(new BoostTargetEffect(7,7, Duration.EndOfTurn));
```

Comment:
You need to change the text on the effect because you're changing the filter text